code/datums/mutations/body.dm

```diff
@@ -165,18 +165,17 @@
 	quality = NEGATIVE
 	mobtypes_allowed = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
 	locked = TRUE //Species specific, keep out of actual gene pool
-	mutadone_proof = TRUE
-	var/original_species = /datum/species/human
+	var/orig_species = /datum/species/human
 
 /datum/mutation/race/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
-	original_species = owner.dna.species
-	. = owner.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE | TR_KEEPAI, FALSE, original_species)
+	orig_species = owner.dna.species
```

Comment:
Keep the current name, use `original_species = src.original_species` in the argument pass, or rename the argument to something like `_original_species`

Reply:
Sure, done.